content/topics/git-basics/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+_db_id: 8
+content_type: topic
+ready: true
+title: Git Basics
+---
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vTfYzPpWfCU3ZD9EPT9PhH-7bDC45fQUz1_YkTJfB80eV1jzAiJ2M55hKJqk59d9hV_k0e6TxO_SLQu/embed?start=false&loop=false&delayms=60000" frameborder="0" width="960" height="569" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```

Comment:
Does topics/git/_index.md cover this info or does it need to be updated? once topics/git/_index.md is updated this be depreciated